website/blog/2023-09-06-Apache-Hudi-From-Zero-To-One-blog-2.mdx

```diff
@@ -0,0 +1,31 @@
+---
+title: "Apache Hudi: From Zero To One (2/10)"
+excerpt: "Dive into read operation flow and query types"
+author: Shiyan Xu
+category: blog
+image: /assets/images/blog/2023-09-06-Apache-Hudi-From-Zero-To-One-blog-2.png
+tags:
+- blog
+- apache hudi
+- query types
+- read operations
```

Comment:
change to `reads`. Lets not introduce word families as tags for the same word. Lets try to keep it to similar tag we have done that already. Pease ensure you always refer to the all tags to find the closest tag that already exists unless absolutely needed.